src/test/java/org/junitpioneer/testkit/assertion/TestCaseAssertBase.java

```diff
@@ -63,7 +63,7 @@ public void whichAborted() {
 	@Override
 	public void withExceptionFulfilling(Predicate<Throwable> predicate) {
 		Throwable thrown = getRequiredThrowable();
-		predicate.test(thrown);
+		assertThat(predicate.test(thrown)).isTrue();
```

Comment:
How about:

```suggestion
		assertThat(predicate).accepts(thrown);
```

Reference: https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/PredicateAssert.html

Reply:
Sounds good. Done.